components/application-authenticators/org.wso2.carbon.identity.application.authenticator.social/src/main/java/org/wso2/carbon/identity/application/authenticator/social/yahoo/YahooOAuth2Authenticator.java

```diff
@@ -220,35 +216,35 @@ protected boolean requiredIDToken(Map<String, String> authenticatorProperties) {
     protected Map<ClaimMapping, String> getSubjectAttributes(OAuthClientResponse token,
                                                              Map<String, String> authenticatorProperties) {
 
-        Map<ClaimMapping, String> claims = new HashMap<ClaimMapping, String>();
+        Map<ClaimMapping, String> claims = new HashMap<>();
 
         try {
-
             String accessToken = token.getParam(OIDCAuthenticatorConstants.ACCESS_TOKEN);
             String url = getUserInfoEndpoint(token, authenticatorProperties);
-
             String json = sendRequest(url, accessToken);
 
             if (!StringUtils.isNotBlank(json)) {
-                log.info("Unable to fetch user claims. Proceeding without user claims");
+                if (log.isDebugEnabled()) {
+                    log.debug("Unable to fetch user claims. Proceeding without user claims");
+                }
                 return claims;
             }
 
             Map<String, Object> jsonObject = JSONUtils.parseJSON(json);
 
-            // Extract the inside profile JSON object.
+            // Extract the inner profile JSON object.
             Map<String, Object> profile = JSONUtils.parseJSON(
                     jsonObject.entrySet().iterator().next().getValue().toString());
 
             if (profile == null) {
-                log.info("Invalid user profile object. Proceeding without user claims");
+                if (log.isDebugEnabled()) {
+                    log.debug("Invalid user profile object. Proceeding without user claims");
```

Comment:
As per security reasons we can't log sensitive data. So in here we can't log "url" or access token. Since user profile object also null, there are no useful information to log with.